db/data_migration/20170818102913_send_missing_people_to_rummager.rb

```diff
@@ -0,0 +1,32 @@
+slugs = %w(
+  nick-hurd
+  nick-gibb
+  damian-green
+  brandon-lewis
+  elizabeth-truss
+  greg-hands
+  claire-perry
+  andrea-leadsom
+  lord-ashton-of-hyde
+  rory-stewart
+  stephen-barclay
+  richard-harrington
+  caroline-nokes
+  lord-oshaughnessy
+  david-rutley
+  rebecca-harris
+  mike-freer
+  nigel-adams
+  stuart-andrew
+  craig-whittaker
+  ian-duncan
+  david-hall--2
+  david-reid
+  duncan-thompson--2
+  david-reed
+  ben-merrick
+)
+
+missing_people = Person.where(slug: slugs)
+
```

Comment:
@leenagupte I don't think a `return` is needed here. If `Person.where(slug: slugs)` returns an empty result set, `missing_people` will be an empty ActiveRecord::Relation which will simply have nothing to iterate on where doing the `.map` - equivalent to `[].each {|x| puts x}` - it just does nothing. The migration will still run without error.